integration-tests/src/test/java/tachyon/underfs/s3/S3UnderStorageCluster.java

```diff
@@ -38,15 +39,14 @@
   public S3UnderStorageCluster(String baseDir, TachyonConf tachyonConf) {
     super(baseDir, tachyonConf);
     mS3Bucket = System.getProperty(INTEGRATION_S3_BUCKET);
-    mBaseDir = mS3Bucket + UUID.randomUUID();
+    mBaseDir = PathUtils.concatPath(mS3Bucket + UUID.randomUUID());
   }
 
   @Override
   public void cleanup() throws IOException {
-    String oldDir = mBaseDir;
-    mBaseDir = mS3Bucket + UUID.randomUUID();
     UnderFileSystem ufs = UnderFileSystem.get(mBaseDir, mTachyonConf);
-    ufs.delete(oldDir, true);
+    ufs.delete(mBaseDir, true);
+    mBaseDir = PathUtils.concatPath(mS3Bucket + UUID.randomUUID());
```

Comment:
I don't think so, `UnderFileSystem.get()` usually only cares about the prefix (definitely true for s3).